README.md

```diff
@@ -291,6 +291,7 @@ ini_setting { '[section1] setting3':
 
 
 ## Reference
+See [REFERENCE.md](https://github.com/puppetlabs/puppetlabs-inifile/blob/master/REFERENCE.md)
 
 ### Public Types
```

Comment:
Reference documentation needs removed as it is now present in REFERENCE.md. Just leave the header and the `See REFERENCE.md` link.